src/main/java/edu/ie3/datamodel/io/naming/DataSourceMetaInformation.java

```diff
@@ -3,16 +3,16 @@
  * Institute of Energy Systems, Energy Efficiency and Energy Economics,
  * Research group Distribution grid planning and operation
 */
-package edu.ie3.datamodel.io.csv;
+package edu.ie3.datamodel.io.naming;
 
 import java.util.Objects;
 import java.util.UUID;
 
-/** Meta information, that can be derived from a certain file name */
-public abstract class FileNameMetaInformation {
+/** Meta information, that can be derived from a data source */
+public abstract class DataSourceMetaInformation {
```

Comment:
I'm not fully convinced with the name... Yes, it does describe a data source, but it does not describe a data source in general. It's more describing a time series data source. But `TimeSeriesDataSourceMetaInformation` is a bit long. Do you have any other suggestion?

Reply:
In that case, how about simply `TimeSeriesMetaInformation`?

Reply:
@johanneshiry